package.json

```diff
@@ -52,7 +52,8 @@
     "test": "npm-run-all dist js-test docs",
     "watch": "npm-run-all --parallel watch-css watch-js",
     "watch-css": "nodemon --ignore js/ --ignore dist/ -e scss -x \"npm run css && npm run css-docs\"",
-    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile-plugins\""
+    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile-plugins\"",
+    "workbox-precache": "workbox-cli inject:manifest --configFile=\"build/workbox.config.json\""
```

Comment:
We need to hook this script in the other scripts, say `docs`. And also rename it to `docs-workbox-precache`.